tests/_support/Step/Acceptance/Installation/Installation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Step\Acceptance\Installation;
+
+use Page\Acceptance\Administrator\LoginPage;
+use Page\Acceptance\Administrator\ControlPanelPage;
+
+class Installation extends \AcceptanceTester
+{
+	/**
+	 * @Given Joomla CMS is installed
+	 */
+	public function joomlaCMSIsInstalled()
+	{
+		$I = $this;
+		// @todo: the following path should be taken from configuration and not hardcoded as it is now.
+		$I->assertFileExists('tests/joomla-cms3/configuration.php');
```

Comment:
I was wondering the same while I was doing it: implement it or remove it.

Okay, I'm removing it in the next update. Thanks